src/omero_cli_render.py

```diff
@@ -35,24 +35,33 @@
 
 from omero import UnloadedEntityException
 
-DESC = {
-    "COPY": "Copy rendering setting to multiple objects",
-    "INFO": "Show details of a rendering setting",
-    "SET": "Set a rendering setting",
-    "EDIT": "Deprecated, please use 'set' instead",
-    "TEST": "Test that underlying pixel data is available"
-}
+HELP = "Tools for working with rendering settings"
 
-HELP = """Tools for working with rendering settings
+INFO_HELP = """Show details of a rendering setting
 
-Examples:
-
-    # %(INFO)s
+    Examples: 
     bin/omero render info RenderingDef:1
     bin/omero render info Image:123
+"""
+
+COPY_HELP = """Copy rendering setting to multiple objects
```

Comment:
I would add: ``Targets can be Dataset(s), Project(s), Plate(s) or Screen(s).

Atm I am not sure if Projects and Screens really can be used. If not true, adjust as needed of course.